diff --git a/themes/bira.zsh-theme b/themes/bira.zsh-theme
index cdae68d..6f1c77b 100644
--- a/themes/bira.zsh-theme
+++ b/themes/bira.zsh-theme
@@ -6,7 +6,7 @@ if [[ $UID -eq 0 ]]; then
     local user_symbol='#'
 else
     local user_host='%{$terminfo[bold]$fg[green]%}%n@%m %{$reset_color%}'
-    local user_symbol='$'
+    local user_symbol='λ'
 fi
 
 local current_dir='%{$terminfo[bold]$fg[blue]%}%~ %{$reset_color%}'
